dlrover/trainer/constants/tf_constants.py

```diff
@@ -62,3 +62,5 @@ class TFConstants(object):
     EstimatorTrainingHooks = Constant("training_hooks")
     EstimatorPredictionHooks = Constant("prediction_hooks")
     EstimatorEvaluationHooks = Constant("evaluation_hooks")
+    EnableEasydl = Constant("enable_easydl")
```

Comment:
It is better to rename `enable_easydl` to `enable_auto_scaling`.  The name "easydl" will be abandoned in DLRover.

Reply:
enable_auto_scaling is added